mlir/test/lit.cfg.py

```diff
@@ -18,7 +18,7 @@
 config.test_exec_root = getattr(config, "quantum_test_dir", ".lit")
 
 # Define substitutions used at the top of lit test files, e.g. %PYTHON.
-python_executable = getattr(config, "python_executable", "python3.10")
+python_executable = getattr(config, "python_executable", "python3.11")
```

Comment:
I'm wondering if this should say "python3" instead, didn't we have a PR a few months ago that already updated this? 🤔

Reply:
Ah yes https://github.com/PennyLaneAI/catalyst/pull/1559/commits/e13325741a3fc422eee0a9404d07b34f193e6b56